src/routes.tsx

```diff
@@ -1,16 +1,16 @@
 import { RouteObject } from 'react-router-dom';
 import { SiteWrapper } from './components/site-wrapper/site-wrapper';
-import { Deleted } from './components/deleted/deleted';
 import { AddNotePage } from './components/add-note-page/add-note-page';
 import { Notes } from './components/notes/notes';
+import { TrashPage } from './components/trash-page/trash-page';
 
 export const routes: RouteObject[] = [
     {
         path: '/',
         element: <SiteWrapper />,
         children: [
             { index: true, element: <Notes /> },
-            { path: 'deleted', element: <Deleted /> },
+            { path: 'trash', element: <TrashPage /> },
```

Comment:
I like old name more

Reply:
By design

Reply:
It's only a title

Reply:
`TrashPage` is more self-explanatory then `deleted`.
I don't want to change it 😇